packages/manager/apps/dedicated/client/app/ip/components/list/byoip/delete/ip-ip-byoip-delete.controller.js

```diff
@@ -0,0 +1,10 @@
+export default /* @ngInject */ ($scope, coreURLBuilder) => {
+  $scope.data = $scope.currentActionData;
```

Comment:
```suggestion
  this.data = $scope.currentActionData;
```
use this inside the controller